README.md

```diff
@@ -39,18 +39,30 @@ built from source using the latest version of [Go](https://golang.org), and the
 available instructions in our
 [Wiki](https://github.com/git-lfs/git-lfs/wiki/Installation#source).
 
-### Usage
+#### Installing from a [binary package](https://github.com/git-lfs/git-lfs/releases)
 
-Git LFS requires a global installation once per-machine. This can be done by
+The included script will install Git LFS binaries. It also runs `git lfs install` to
+perform required global configuration changes.
+
+```bash
```

Comment:
It would be preferred to use `ShellSession` as the language for this, which looks like:

(With `bash`):
```bash
$ ./install.sh
```

(With `ShellSession`):
```ShellSession
$ ./install.sh
# ...
```

The two look identical without a line below, but it will make the diff easier to grok in the future if someone does wish to add output below `$ ./install.sh`.